daemon/src/projection.rs

```diff
@@ -979,26 +979,26 @@ pub struct CfdOrder {
     pub creation_timestamp: Timestamp,
     pub settlement_time_interval_in_secs: u64,
 
-    /// Margin per parcel from the perspective of the role
+    /// Margin per lot from the perspective of the role
     ///
     /// Since this is a calculated value that we need in the UI this value is based on the
     /// perspective the role (i.e. taker/maker)
     #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
-    pub margin_per_parcel: Amount,
+    pub margin_per_lot: Amount,
 
-    /// Initial funding fee per parcel from the perspective of the role
+    /// Initial funding fee per lot from the perspective of the role
     ///
     /// Since this is a calculated value that we need in the UI this value is based on the
     /// perspective the role (i.e. taker/maker)
     #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
-    pub initial_funding_fee_per_parcel: SignedAmount,
```

Comment:
Are any of these fields used by `autotaker` or other HTTP clients? If yes, then we might want to consider duplicating them with a new field name first and update all clients before we remove them.

Reply:
great idea. I will do that before I merge this PR - I posted this first to get the buy-in from everyone first.

Reply:
I have cheched the autotaker and automation, none of them care about parcel so far :)